lib/timex.ex

```diff
@@ -1202,7 +1202,7 @@ defmodule Timex do
         def day_to_num(unquote(atom)), do: unquote(day_num)
       end
 
-    Module.eval_quoted(__MODULE__, day_quoted, [], __ENV__)
+    Code.eval_quoted(day_quoted, [], __ENV__)
```

Comment:
`Code.eval_quoted` exists in very early versions of Elixir, so this shouldn't impact the minimum Elixir version required for this library.